corehq/apps/smsbillables/models.py

```diff
@@ -225,6 +226,11 @@ def get_by_criteria(cls, direction, domain=None):
         return cls.objects.filter(criteria=criteria.id).latest('date_created')
 
 
+@quickcache(['sms_backend_id'])
+def _sms_backend_is_global(sms_backend_id):
+    return SMSBackend.get(sms_backend_id).is_global
```

Comment:
@biyeun / @gcapalbo do we worry about this value being stale? My intuition was a resounding *no*, because `is_global` seems like the type of thing that would be set once when the backend is added to the db, and never change again, or at least only change in rare circumstances, it seems fine even if it takes 10 minutes to go into effect.